docs/user/vsphere/install_upi.md

```diff
@@ -180,9 +180,70 @@ test-vsphere
 
 NOTE: The filename for `install-config` in the `INSTALL_DIR` must be `install-config.yaml`
 
+### Invoking the installer to get manifests
+Given that you have setup the `INSTALL_DIR` with the appropriate `install-config.yaml`, you can create manifests by using the `create manifests` target. For example,
+
+```console
```

Comment:
Maybe truncated find if the items to sure that this is an example as there list might change.. keep the openshift/ machine objects

Reply:
@abhinavdahiya  Updated but not sure if this is what you had in mind. If not, let me know and I am happy to revise.